Server/package.json

```diff
@@ -14,5 +14,7 @@
     "express": "^4.17.1",
     "mongoose": "^5.6.0"
   },
-  "devDependencies": {}
+  "devDependencies": {
+    "cors": "^2.8.5"
```

Comment:
why in dev dependencies and not in dependencies?

Reply:
Moved to dependencies in base branch.